<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_volume</class>
 <widget class="QWidget" name="fm_volume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="gb_volume_resizing">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="hs_width_volume">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="sb_width_volume">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="hs_height_volume">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sb_height_volume">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Length:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="hs_depth_volume">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="sb_depth_volume">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Resolution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rb_lowresolution">
          <property name="text">
           <string>Regular</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_mdresolution">
          <property name="text">
           <string>Good</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_hgresolution">
          <property name="text">
           <string>Better</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hs_width_volume</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sb_width_volume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hs_depth_volume</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sb_depth_volume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hs_height_volume</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sb_height_volume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_width_volume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hs_width_volume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_height_volume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hs_height_volume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_depth_volume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hs_depth_volume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
